german/words/an.md

```diff
@@ -0,0 +1,28 @@
+# an 
+## Type 
+- _**Präposition**_ 
+- _**Adverb**_ 
+## Pronunciation 
+- _**an**_ 
+## Meaning 
+- **''Präposition des Ortes; mit dem Akkusativ (zur Anzeige der Richtung):'' bestimmt ein Ziel, einen Zielpunkt, mit welchem etw. in Berührung kommt** 
+- **''Präposition des Ortes; mit dem Dativ (zur Anzeige der Lage):'' bestimmt einen Ort, mit dem etw. in Berührung ist** 
+- **''Temporale Präposition; mit dem Dativ:'' bestimmt einen Zeitpunkt** 
+- **''Objekt-Präposition (mit bestimmten Verben); mit dem Akkusativ:'' zeigt eine virtuelle Richtung an** 
+- **''[[umgangssprachlich]]:'' [[einschalten|eingeschaltet]] oder [[anzünden|angezündet]]** 
+## Examples 
+- **Er hängte das Bild ''an'' die Wand.** 
+- **Das Bild hing ''an'' der Wand.** 
+- **Er wird ihn erst ''am'' Sonntag wieder sehen.** 
+- **Jedes Jahr ''an'' Weihnachten bekommt er große Geschenke.** 
+- **''An'' Samstagen war immer besonders viel los.** 
+- **Sie dachte ''an'' ihren Freund.** 
+- **Horst glaubt ''an'' den Weihnachtsmann.** 
+- **Das Licht ist noch ''an!''** 
+- **Du hast die Kerzen ''an'' gelassen!** 
+## Translations 
+- **en: to, on** 
+- **es: en, domingo, por, navidad** 
```

Comment:
this is a bug, as `navidad` is not a translation of `an`. Fixed in https://github.com/qu1queee/commongermanwords/commit/9017a6c9a24ac4abfd45cfa1f22a705637041fb8